File: files/samples/System_Bar_Colors.py
Translated code:

```python
import native

def OnStart():
    native.app.SetStatusBarColor("#ff0000")
    native.app.SetNavBarColor("#ff0000")
    
    lay = native.app.CreateLayout("linear", "VCenter,FillXY")
    
    txt = native.app.CreateText("Hello")
    txt.SetTextColor("#ff0000")
    txt.SetTextSize(32)
    lay.AddChild(txt)
    
    native.app.AddLayout(lay)
```

File: files/samples/js/System_Bar_Colors.js

//Called when application is started.
function OnStart()
{
    //Set status and nav bar colors.
	//(Works on Lollipop or newer.)
    app.SetStatusBarColor( "#ff0000" )
    app.SetNavBarColor( "#ff0000" )
    
	//Create a layout with objects vertically centered.
	lay = app.CreateLayout( "linear", "VCenter,FillXY" )	

	//Create a text label and add it to layout.
	txt = app.CreateText( "Hello" )
	txt.SetTextColor( "#ff0000" )
	txt.SetTextSize( 32 )
	lay.AddChild( txt )
	
	//Add layout to app.	
	app.AddLayout( lay )
}

